content/blog/wingingit/winging-it-01.md

```diff
@@ -0,0 +1,39 @@
+---
+title: 1. CSS Cascade Layers
```

Comment:
Right now the episode numbers seem a bit strange out of context. I could see a few options, like adding the word 'episode' (eg `CSS Cascade Layers (episode 1)`)? Or putting the episode number in the subtitle? Or making that it's own metadata, that only shows sometimes - like with the new `length` data?

Reply:
@mirisuzanne We discussed the episode name in the post title... but I like the idea of having its own metadata.  https://github.com/oddbird/oddleventy/pull/378/files/e6d110695220449656fc83f90e5d2f7b15f472ab#r1260210525

Reply:
@jgerigmeyer following up from our discussion during stand up, my proposal for the winging it specific byline is/was:

Episode Title
Winging It episode # _hosted by_ name, name and name _on_ July 20, 2023 | 60 mins

but now that @mirisuzanne mentioned in the review response that we could treat all podcasts, not our own in a similar way, then I think episode number wouldn't work as I am willing to bet that not all podcasts include a number. We could drop it in favor of something like:

Episode Title
_on_ Winging It _with (or featuring?)_ name, name and name _on (or posted?)_ July 20, 2023 | 60 mins

Reply:
Added in https://github.com/oddbird/oddleventy/commit/09980562e9ee0dfae022d5464de9f84bd7e54405